component-catalog/src/Examples/FocusRing.elm

```diff
@@ -47,11 +50,48 @@ example =
         , example_ FocusRing.tightStyles
         ]
     , about =
-        [ Text.smallBody
+        [ Text.mediumBody
```

Comment:
Should this be smallBody?

Reply:
Oh I don't know actually! I wasn't sure when we were using one vs. the other. Feel free to change!